molecule_libvirt/playbooks/templates/vm/vm.xml.j2

```diff
@@ -0,0 +1,96 @@
+<!--
+WARNING: THIS IS AN AUTO-GENERATED FILE. CHANGES TO IT ARE LIKELY TO BE
+OVERWRITTEN AND LOST. Changes to this xml configuration should be made using:
+  virsh edit rhel7.4
+or other application using the libvirt API.
+-->
+
+<domain type='kvm'>
+  <name>{{ item.name }}</name>
+  <uuid>{{ lookup('lines','uuidgen') }}</uuid>
+  <memory unit='GiB'>{{ memory_size | default(default_memory_size) }}</memory>
+  <currentMemory unit='GiB'>{{ memory_size | default(default_memory_size) }}</currentMemory>
+  <vcpu placement='static'>{{ vcpu | default(default_vcpu) }}</vcpu>
+  <os>
+    <type arch='x86_64' machine='pc'>hvm</type>
+    <boot dev='hd'/>
+  </os>
+  <features>
+    <acpi/>
+    <apic/>
+    <vmport state='off'/>
+  </features>
+  <cpu mode='custom' match='exact' check='none'>
+    <model fallback='forbid'>qemu64</model>
```

Comment:
I wonder if that is what may break fedora zuul job if nested virtualization is not allowed on that nodes.

Reply:
done. I will update the doc soon.

Reply:
- The README has been update.
- fallback action is  `allow` now. 
- CPU model is now a variable.